code/maps/map_prep.R

```diff
@@ -4,79 +4,150 @@ library(readxl)
 library(tigris)
 library(sf)
 library(sp)
+library(rgeos)
 
 setwd("C:/Users/kathy/Documents/github/CARES")
 
 ############################
 # read in data
 ############################
-# cleaned ZCTA-level PPP dataset
-ppp <- read_csv("data/tidy_data/Michigan.csv")
-
-# ZCTA to block-group mapping
-zcta_mapping <- read_csv("data/Lookup Tables/acs_2018_block_groups_zcta.csv",
-                         col_types = cols(
-                             .default = col_character(),
-                             block_group_interpolated_latitude = col_double(),
-                             block_group_interpolated_longitude = col_double(),
-                             zcta_interpolated_latitude = col_double(),
-                             zcta_interpolated_longitude = col_double()
-                         ))
-
-# Demographic files
-zcta_demo <- read_csv("data/demographics/zipcodeDemographics.csv")
-county_demo <- read_csv("data/demographics/countyDemographics.csv")
-cd_demo <- read_csv("data/demographics/congressional districtDemographics.csv")
-state_demo <- read_csv("data/demographics/stateDemographics.csv")
-
-# download shapefiles
-#zcta_shp <- zctas(state = "MI")
-counties_shp <- counties(state = "MI")
-#cd_shp <- congressional_districts()
+# create base ppp dataset
+source("bin/ppp_data_merge.R")
```

Comment:
This is where adbs is coming from, right?

The end result shouldn't be any different, but we could also run the congressional district script here to pull the demographics data with the boundaries attached (and comment out the sample plots at the bottom)..

Reply:
It would cut out some steps here, but also might be less efficient since the census call takes quite a while.

Reply:
Ah ok, we could do that. I assumed demographics.R would be run separately to obtain the demographics data as csv files first, so in this script I'm just downloading the boundary files using the tigris package and reading in the demographics csv's. We could have all the data download in a separate file if that keeps the pipeline cleaner.